front/pages/api/auth/[...nextauth].js

```diff
@@ -8,14 +8,17 @@ export const authOptions = {
       clientId: process.env.GITHUB_ID,
       clientSecret: process.env.GITHUB_SECRET,
     }),
-    // GoogleProvider({
-    //   clientId: process.env.GOOGLE_CLIENT_ID,
-    //   clientSecret: process.env.GOOGLE_CLIENT_SECRET,
-    // }),
+    GoogleProvider({
+      clientId: process.env.GOOGLE_CLIENT_ID,
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET,
+    }),
   ],
   secret: process.env.NEXTAUTH_SECRET,
   callbacks: {
     session: async ({ session, token, user }) => {
+      // console.log("TOKEN", token);
```

Comment:
nit 🧹

Reply:
I keep them around as I they are useful for debugging when working with auth providers.